.storybook/config.js

```diff
@@ -1,6 +1,9 @@
-import { configure } from '@storybook/react';
+import {configure, addDecorator} from '@storybook/react';
+import {withKnobs} from '@storybook/addon-knobs';
 import '../momentum-ui.scss';
 
+addDecorator(withKnobs);
```

Comment:
Being nit-picky here, but can you add the decorator just before the configure call?
As we might add more add-ons, it feels more natural (to me) for them to be close the the configuration return